sdk/core/azure-core/inc/azure/core/operation_status.hpp

```diff
@@ -24,28 +24,28 @@ namespace Azure { namespace Core {
 
   public:
     /**
-     * @brief Construct a #OperationStatus with \p value.
+     * @brief Construct an #OperationStatus with \p value.
      *
      * @param value A non-absent value to initialize with.
```

Comment:
Apply the same fix to other uses of `non-absent`.
```suggestion
     * @param value The value to initialize with.
```